src/components/App.js

```diff
@@ -31,6 +32,7 @@ const App = () => (
         <Route path="/availablementors" component={AvailableMentors} />
         <Route path="/password-reset" component={PasswordReset} />
         <Route path="/settings" component={Settings} />
+        <Route path="/nofound" component={NoFound} />
```

Comment:
Here we can use  ```<Route path="*" component={NoFound} />``` or  ```<Route component={NoFound} />``` to show the 404 page when the route doesn't have a match.

Reply:
I love the design ❤️ 😄